src/Map/Map.jsx

```diff
@@ -290,13 +329,19 @@ export class Map extends Component {
   }
 
   render () {
-    const {center, mapOptions, tileLayerOptions, wms, zoomControlPosition} = this.props
+    const {
+      center,
```

Comment:
couldn't this be 

```javascript
const {
  center = this.defaultCenter(),
  mapOptions,
  //...
}
```

and then you don't have to have the short-circuit || in the return?